static/gsApp/views/subscriptionPage/usageTotalsTable.tsx

```diff
@@ -199,38 +200,40 @@ function UsageTotalsTable({category, isEventBreakdown, totals, subscription}: Pr
           category={category}
           totals={totals}
         />
-        <OutcomeSection
-          isEventBreakdown={isEventBreakdown}
-          name={totalDropped}
-          quantity={totals.dropped}
-          category={category}
-          totals={totals}
-        >
-          <OutcomeRow
-            indent
-            name={t('Over Quota')}
-            quantity={totals.droppedOverQuota}
-            category={category}
-            totals={totals}
-          />
-          <OutcomeRow
-            indent
-            name={t('Spike Protection')}
-            quantity={totals.droppedSpikeProtection}
-            category={category}
-            totals={totals}
-          />
-          <OutcomeRow
-            indent
-            name={t('Other')}
-            quantity={totals.droppedOther}
+        {!isSeer(category) && (
```

Comment:
could we make this something like `showAllTotals` and make it a parameter for UsageTotalsTable? then in the parent component for UsageTotals/CombinedUsageTotals add the isSeer check